test/common.js

```diff
@@ -20,6 +20,13 @@ if (process.env.TEST_THREAD_ID) {
 }
 exports.tmpDir = path.join(exports.testDir, exports.tmpDirName);
 
+// Create tmpDir if it does not exist
+try {
+  fs.accessSync(exports.tmpDir);
```

Comment:
Why not just `try { fs.mkdirSync(exports.tmpDir); }`? You don't need the `accessSync()` call at all then. And `mkdirSync()` being in the `catch` block is wrong because it can still throw.